app/modules/annotations/models.py

```diff
@@ -119,3 +121,14 @@ def check_job_status(self, job_id):
         # TODO Poll ACM to see what's happening with this job, if it's ready to handle and we missed the
         # response, process it here
         return True
+
+    def set_bounds(self, bounds):
+        self.validate_bounds(bounds)
+        self.bounds = bounds
+
+    @classmethod
+    def validate_bounds(cls, bounds):
+        assert isinstance(bounds, dict)
```

Comment:
Style question. This is externally provided information. 
My C brain always says that assert should be used for things that indicate a coding error in our thing (houston).
This implies an error in a calling thing.
I've been implementinng these as HoustonException and setting the response info returned. This feels less controlled. 
But this may well be me trying to impose C logic here. Thoughts?

Reply:
really, i just threw this in as a whim.  right now we are leaving `bounds` open to be a fairly flexible json structure as a way to future-proof what a "shape" of an annotation looks like.  i just wanted one place to pass this through -- which is why i also imagined it being used by `set_bounds()` for example.

as for the `abort()` inside schema, i was again kind of mirroring what i saw being done in users/parameters.py ... i am definitely open to thinking about ways to modify this code that focuses this more on keeping it contained within parameters.py .... likewise, even within the `replace()` method (for patch) in annotations/parameters.py i do the `abort()` there... which i did rather than simply return `False` (which i believe, ultimately, would still throw the same 422) just for the sake of giving a more detailed message, as the _generic_ 422 message is vague to say the least.